cmd/main.go

```diff
@@ -88,6 +88,7 @@ func init() {
 	flag.StringVar(&edgelbOptions.Host, "edgelb-host", constants.DefaultEdgeLBHost, "the host at which the edgelb api server can be reached")
 	flag.BoolVar(&edgelbOptions.InsecureSkipTLSVerify, "edgelb-insecure-skip-tls-verify", false, "whether to skip verification of the tls certificate presented by the edgelb api server")
 	flag.StringVar(&edgelbOptions.Path, "edgelb-path", constants.DefaultEdgeLBPath, "the path at which the edgelb api server can be reached")
+	flag.IntVar(&translatorapi.DefaultEdgeLBPoolSize, "edgelb-default-pool-size", constants.DefaultEdgeLBPoolSize, "the default number of load balancer instances in the edgelb pools")
```

Comment:
Why is this not set in `edgelbOptions` like other flag values? Then you can leave default as `int32` and not change the rest of the code.

Reply:
Because it would require a deep refactor of dklb internals to get `edgelbOptions` available [here](https://github.com/mesosphere/dklb/blob/master/pkg/translator/api/base.go#L60-L62).

Reply:
OK